trident-concepts/snapshots.adoc

```diff
@@ -13,6 +13,7 @@ summary: Learn more about how Astra Trident handles the creation of volume snaps
 Learn more about how Astra Trident handles the creation of volume snapshots for its drivers.
 
 * For the `ontap-nas`, `ontap-san`, `aws-cvs`, `gcp-cvs`, and `azure-netapp-files` drivers, each Persistent Volume (PV) maps to a FlexVol. As a result, volume snapshots are created as NetApp snapshots. NetApp’s snapshot technology delivers more stability, scalability, recoverability, and performance than competing snapshot technologies. These snapshot copies are extremely efficient both in the time needed to create them and in storage space.
+* For the `ontap-nas-flexgroup` driver, each Persistent Volume (PV) maps to a FlexGroup. As a result, volume snapshots are created as NetApp FlexGroup snapshots. NetApp’s snapshot technology delivers more stability, scalability, recoverability, and performance than competing snapshot technologies. These snapshot copies are extremely efficient both in the time needed to create them and in storage space.
```

Comment:
@juliantap, after reviewing the FlexGroup TR these snapshots are also called NetApp snapshots. They behave the same as other snapshots and the only distinction is that they map to the FlexGroup instead of the Flexvol.